File: 10871.c
#include<stdio.h>

int main(){
    int lengthOfInput, X;
    scanf("%d %d", &lengthOfInput, &X);
    
    for(int i = 0; i < lengthOfInput; i++){
        int currentInput;
        scanf("%d", &currentInput);
        if(currentInput < X)
            printf("%d ", currentInput);
    }
    return 0;
}

File: 10871.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int lengthOfInput = sc.nextInt();
        int X = sc.nextInt();
        
        for(int i = 0; i < lengthOfInput; i++){
            int currentInput = sc.nextInt();
            if(currentInput < X)    System.out.print(currentInput+" ");
        }
    }
}
